test/SlowTests/Bugs/Indexing/DynamicFields.cs

```diff
@@ -99,7 +99,7 @@ select new
 
         [Theory]
         [RavenData(SearchEngineMode = RavenSearchEngineMode.Lucene)]
-        [RavenData(SearchEngineMode = RavenSearchEngineMode.Corax, Skip = "RavenDB-17966")]
+        [RavenData(SearchEngineMode = RavenSearchEngineMode.Corax, Skip = "Lucene API for create field")]
```

Comment:
"Lucene API for create field" - what does it mean?

Reply:
https://github.com/ravendb/ravendb/blob/cd0ce04c44454272d85ccf75782a6be6b3bff7fc/test/SlowTests/Bugs/Indexing/DynamicFields.cs#L47-L58

The index creates a field via Lucene API, without our `CreateField()` wrapper.

Reply:
ok